src/main/groovy/com/ullink/Msbuild.groovy

```diff
@@ -73,6 +73,9 @@ class Msbuild extends ConventionTask {
     ProjectFileParser getMainProject() {
         if (resolveProject()) {
             projectParsed
+        } else {
+            logger.warn "Main project was resolved to null due to a parse error. The .sln file might be missing or incorrectly named."
```

Comment:
I think we should tell about the project file also, not only the sln might be wrong :)

And maybe let's put the actual name we looked for, so that it's more clear (in the exception). The "matches the one of the repository" isn't valid always for other people using this lib.